tests/test_exotic_operation_combos.py

```diff
@@ -57,8 +57,9 @@ def test_replace_deleted_object_with_update(self):
 
         self.session.delete(article)
         self.session.flush()
-
-        article2.id = article.id
+        # we were earlier updating ID which didn't seem right, so changed this to name since
+        # id is used by us for identity in operation
+        article2.name = article.name
```

Comment:
Actually this seems to be the reason for this testcase if you see the skipif - it is trying to update the "identity" and seeing how that works

Reply:
so that skipif was added by us only, when we were extending package support to mssql db,  i changed it because changing ID won't be right in real sense and i tried it out as well
.
say i change the ID of `article2` to `article` now if i try to collect version objects of article2 i'll get version object of article1
.
i missed removing the skipif, i'll remove the skipif since we no longer update the identity so this should not fail with mssql.